### PR TITLE
CLOUD-625 - add chaos mesh tests into cloud pipelines

### PR DESCRIPTION
**Pull request by tplavcic**
Depends on: https://github.com/percona/percona-server-mongodb-operator/pull/669

For some reason self-healing tests were not added into openshift pipelines, so I didn't add these new also.

### CODE REVIEW

File: cloud/jenkins/psmdb_operator_gke_version.groovy [hunk 1]
```diff
@@ -290,7 +290,9 @@ pipeline {
                         CreateCluster('selfhealing')
                         runTest('storage', 'selfhealing')
                         runTest('self-healing', 'selfhealing')
+                        runTest('self-healing-chaos', 'selfhealing')
                         runTest('operator-self-healing', 'selfhealing')
+                        runTest('operator-self-healing-chaos', 'selfhealing')
                         ShutdownCluster('selfhealing')
```
Comment (cap1984) [inline cloud/jenkins/psmdb_operator_gke_version.groovy:296]: no self-healing-advanced-chaos ?

File: cloud/jenkins/psmdb_operator_eks.groovy [hunk 2]
```diff
@@ -254,7 +254,9 @@ EOF
             steps {
                 runTest('storage')
                 runTest('self-healing')
+                runTest('self-healing-chaos')
                 runTest('operator-self-healing')
+                runTest('operator-self-healing-chaos')
```
Comment (cap1984) [inline cloud/jenkins/psmdb_operator_eks.groovy:259]: no self-healing-advanced-chaos ?

File: cloud/jenkins/pxc_operator_minikube.groovy [hunk 3]
```diff
@@ -267,7 +267,9 @@ pipeline {
                     runTest('affinity')
                     runTest('one-pod')
                     runTest('self-healing-advanced')
+                    runTest('self-healing-advanced-chaos')
                     runTest('operator-self-healing')
+                    runTest('operator-self-healing-chaos')
```
Comment (cap1984) [inline cloud/jenkins/pxc_operator_minikube.groovy:272]: no self-healing-advanced-chaos ?

**Review by cap1984 — COMMENTED** [review]
why not all

File: cloud/jenkins/psmdb_operator_gke_version.groovy (hunk 1 above)
Comment (tplavcic) [reply cloud/jenkins/psmdb_operator_gke_version.groovy:296]: it seems that for PSMDB that test doesn't exist

File: cloud/jenkins/psmdb_operator_eks.groovy (hunk 2 above)
Comment (tplavcic) [reply cloud/jenkins/psmdb_operator_eks.groovy:259]: it seems that for PSMDB that test doesn't exist

File: cloud/jenkins/pxc_operator_minikube.groovy (hunk 3 above)
Comment (tplavcic) [reply cloud/jenkins/pxc_operator_minikube.groovy:272]: `self-healing-advanced-chaos` is added above

**Review by cap1984 — APPROVED** [review]

**Pull request merged**